apps/wallet/src/types/node-env.d.ts

```diff
@@ -3,6 +3,6 @@
 
 declare namespace NodeJS {
     interface ProcessEnv {
-        readonly NODE_ENV: 'development' | 'production' | undefined;
+        readonly NODE_ENV: 'development' | 'production' | 'test' | undefined;
```

Comment:
this is a global override, and wallet does not have `skipLibCheck` enabled, so it type-checks the SDK, which now checks `process.env.NODE_ENV === 'test'`.  With the old definition, this isn't valid, so it was breaking the wallet build